src/pages/Donate.tsx

```diff
@@ -0,0 +1,234 @@
+        [theme.breakpoints.up("sm")]: {
+            paddingTop: "2rem"
+        }
+    },
+    root: {
+        flexGrow: 1,
+        justifyContent: 'center',
+        backgroundColor: 'white',
+        opacity: '0.75', 
+        [theme.breakpoints.up("xs")]: {
+            paddingTop: "1rem"
+        }
+    },
+    box: {
+        padding: theme.spacing(2),
+        textAlign: 'center',
+        variant: 'outlined',
+        fontSize: "16px",
+    },
+    circle: {
+        border: "0.1em solid #233768",
+        borderRadius: "100%",
+        height: "1.6em",
+        width: "1.6em",
+        textAlign: "center",
+        marginTop: "0.10em",
+        marginBottom: "0.10em",
+        fontSize: "1.5em",
+        fontWeight: "bold",
+        fontFamily: "sans-serif",
+        color: "#233768",
+        margin: "47%"
+    },
+    paypalButton: {
+        margin: theme.spacing(2),
+        backgroundColor: "white"
+    },
+    cashButton: {
+        margin: theme.spacing(2),
+        backgroundColor: "white"
+    },
+
+    buttonContainer: {
+        textAlign: "center"
+    }
+
+
+}));
+
+function Donate() {
+    const styles = useStyles();
+    return (
+        <>
+            <NavBar />
+            <div className={styles.wrapper}>
+                <div className={styles.container}>
+                    <div className={styles.titleWrapper}>
+                        <Typography
+                            variant="h3"
+                            component="h1"
+                            align="center"
+                            color="primary"
+                        >
+                            Donate
+                        </Typography>
+                    </div>
+                </div>
+                <div className={styles.container}>
+                    <Typography
+                        variant="body1"
+                        component="p"
+                        align="center"
+                        color="secondary"
+                    >
+                        If you'd like to donate funds to our cause, we would apply your kind donations towards
+                    </Typography>
+                    <Typography
+                        variant="body1"
+                        component="p"
+                        align="center"
+                        color="secondary"
+                    >
+                        keeping the platform up and running in the following ways:
+                    </Typography>
+                    <br></br>
+                </div>
+                <div className={styles.gridContainer}>
+                    <Grid container className={styles.root}>
+                        <Grid item xs={12} md={3}>
+                            <div className={styles.circle}>1</div>
+                            <Box className={styles.box}>
+                                Site hosting and server maintenance
+                            </Box>
+                        </Grid>
```

Comment:
predfine in separate const, to avoid code duplication (see comment below for ex)